app/src/ui/changes/changes-list.tsx

```diff
@@ -560,13 +560,38 @@ export class ChangesList extends React.Component<
       { type: 'separator' },
     ]
     if (paths.length === 1) {
+      const shouldEnableContextMenu = Path.basename(path) !== GitIgnoreFileName
       items.push({
         label: __DARWIN__
           ? 'Ignore File (Add to .gitignore)'
           : 'Ignore file (add to .gitignore)',
         action: () => this.props.onIgnoreFile(path),
-        enabled: Path.basename(path) !== GitIgnoreFileName,
+        enabled: shouldEnableContextMenu,
       })
+
+      const pathComponents = path.split(Path.sep)
+      if (pathComponents.length > 0) {
```

Comment:
```suggestion
      if (pathComponents.length > 1) {
```
Otherwise we get:
![Showing an empty menu item.](https://github.com/desktop/desktop/assets/75402236/94699f69-ac85-4251-8a81-619d526f5c6f)